lib/ocpp/v201/charge_point.cpp

```diff
@@ -424,6 +424,17 @@ void ChargePoint::clear_customer_information(const std::optional<CertificateHash
     }
 }
 
+void ChargePoint::configure_message_logging_callback(const std::string& message_log_path) {
```

Comment:
I like that you moved it to a function, cleans it up a bit. The only thing is I don't see how you are configuring a callback.